LICENSE-3rdparty.csv

```diff
@@ -1,1408 +1,1408 @@
-core,"k8s.io/kubernetes/pkg/kubelet/secret",Apache-2.0
-core,"k8s.io/kubernetes/pkg/kubelet/types",Apache-2.0
-core,"k8s.io/kubernetes/pkg/kubelet/util",Apache-2.0
-core,"k8s.io/kubernetes/pkg/kubelet/util/format",Apache-2.0
-core,"k8s.io/kubernetes/pkg/kubelet/util/manager",Apache-2.0
-core,"k8s.io/kubernetes/pkg/proxy/util",Apache-2.0
-core,"k8s.io/kubernetes/pkg/util/hash",Apache-2.0
-core,"k8s.io/kubernetes/pkg/util/parsers",Apache-2.0
-core,"k8s.io/kubernetes/pkg/util/sysctl",Apache-2.0
-core,"k8s.io/kubernetes/pkg/volume",Apache-2.0
-core,"k8s.io/kubernetes/pkg/volume/util/fs",Apache-2.0
-core,"k8s.io/kubernetes/pkg/volume/util/fsquota",Apache-2.0
-core,"k8s.io/kubernetes/pkg/volume/util/fsquota/common",Apache-2.0
-core,"k8s.io/kubernetes/pkg/volume/util/hostutil",Apache-2.0
-core,"k8s.io/kubernetes/pkg/volume/util/recyclerclient",Apache-2.0
-core,"k8s.io/kubernetes/pkg/volume/util/subpath",Apache-2.0
-core,"k8s.io/kubernetes/third_party/forked/golang/expansion",BSD-3-Clause
-core,"k8s.io/metrics/pkg/apis/custom_metrics",Apache-2.0
-core,"k8s.io/metrics/pkg/apis/custom_metrics/install",Apache-2.0
-core,"k8s.io/metrics/pkg/apis/custom_metrics/v1beta1",Apache-2.0
-core,"k8s.io/metrics/pkg/apis/custom_metrics/v1beta2",Apache-2.0
-core,"k8s.io/metrics/pkg/apis/external_metrics",Apache-2.0
-core,"k8s.io/metrics/pkg/apis/external_metrics/install",Apache-2.0
-core,"k8s.io/metrics/pkg/apis/external_metrics/v1beta1",Apache-2.0
-core,"k8s.io/mount-utils",Apache-2.0
-core,"k8s.io/utils/buffer",Apache-2.0
-core,"k8s.io/utils/exec",Apache-2.0
-core,"k8s.io/utils/integer",Apache-2.0
-core,"k8s.io/utils/io",Apache-2.0
-core,"k8s.io/utils/keymutex",Apache-2.0
-core,"k8s.io/utils/net",Apache-2.0
-core,"k8s.io/utils/nsenter",Apache-2.0
-core,"k8s.io/utils/path",Apache-2.0
-core,"k8s.io/utils/pointer",Apache-2.0
-core,"k8s.io/utils/trace",Apache-2.0
-core,"sigs.k8s.io/apiserver-network-proxy/konnectivity-client/pkg/client",Apache-2.0
-core,"sigs.k8s.io/apiserver-network-proxy/konnectivity-client/proto/client",Apache-2.0
-core,"sigs.k8s.io/controller-runtime/pkg/scheme",Apache-2.0
-core,"sigs.k8s.io/structured-merge-diff/v4/fieldpath",Apache-2.0
-core,"sigs.k8s.io/structured-merge-diff/v4/merge",Apache-2.0
-core,"sigs.k8s.io/structured-merge-diff/v4/schema",Apache-2.0
-core,"sigs.k8s.io/structured-merge-diff/v4/typed",Apache-2.0
-core,"sigs.k8s.io/structured-merge-diff/v4/value",Apache-2.0
-core,"sigs.k8s.io/yaml",MIT
+Component,Origin,License,Copyright
+core,cloud.google.com/go,Apache-2.0,UNKNOWN
+core,cloud.google.com/go/compute/metadata,Apache-2.0,UNKNOWN
+core,cloud.google.com/go/iam,Apache-2.0,UNKNOWN
+core,cloud.google.com/go/internal,Apache-2.0,UNKNOWN
+core,cloud.google.com/go/internal/optional,Apache-2.0,UNKNOWN
+core,cloud.google.com/go/internal/trace,Apache-2.0,UNKNOWN
+core,cloud.google.com/go/internal/version,Apache-2.0,UNKNOWN
+core,cloud.google.com/go/storage,Apache-2.0,UNKNOWN
+core,code.cloudfoundry.org/bbs,Apache-2.0,UNKNOWN
```

Comment:
These should all be: CloudFoundry.org Foundation, Inc
https://github.com/cloudfoundry/bbs/blob/master/NOTICE#L1